### ISSUE TITLE
Use dynamic links to alerting drivers services instead of hardcoded links

### ISSUE THREAD

**Issue by catherineluse** [rancher/dashboard#5713]
This is a follow-up to feedback on my PR related to project alerting. Sean mentioned that these are basically magic URLs:

```
export const MS_TEAMS_URL = 'http://rancher-alerting-drivers-prom2teams.cattle-monitoring-system.svc:8089/v2/connector';
```
He said we should organize URLs in a place that makes more sense, such as environment variables. https://github.com/Sean-McQ/dashboard/pull/1#discussion_r856617901

**Comment by catherineluse** [comment]
Update: We are planning to change this so that instead of showing the same hardcoded URL for everyone, we will dynamically check to see if the Rancher alerting drivers service is installed.

If the service exists, we will provide a link to the actual service that exists, which may be in a different namespace than the hardcoded link. This enables project owners to have access to the functionality in spite of not having access to the `cattle-monitoring-system` namespace.

If the service does not exist, we won't provide a URL.

**Comment by gaktive** [comment]
Can document the behaviour for 2.6.5 for now and fix in 2.6.6+

**Comment by gaktive** [comment]
@catherineluse to sync with @btat. There's one field that you can't add text to and there's one link that brings up a blank page (items to be filed).

**Comment by catherineluse** [comment]
Update: For the field you can't add text to, I couldn't reproduce the error. And there is a bug fix for the error about the blank page that was just merged. So I'd say that doesn't need to be called out after all.

**Comment by catherineluse** [comment]
Can't work on this until a routing bug is fixed

**Comment by catherineluse** [comment]
This is unblocked now.

Also, if we want to officially support the alerting drivers for project monitoring, we have to document that you will need to use the `fullNameOverride` value to prevent a naming conflict with any other installations of the helm chart that are also in the same cluster.

Arvind said:

> I think you can resolve it by providing fullnameOverride to the values.yaml of the deployed chart (both to the main chart and the subcharts), that should create these resources using the provided release name instead

**Comment by catherineluse** [comment]
I'm thinking this could be moved to the icebox because it was never officially confirmed that the alerting drivers are supported for project monitoring. Until that is confirmed, I think other work would take higher priority.

**Comment by catherineluse** [comment]
Closing as a duplicate of https://github.com/rancher/dashboard/issues/2651
